velox/functions/prestosql/ArithmeticImpl.h

```diff
@@ -44,10 +44,15 @@ round(const TNum& number, const TDecimals& decimals = 0) {
   }
 
   double factor = std::pow(10, decimals);
+  double variance = 0.1;
```

Comment:
Can you explain this value? Why 0.1? 

Use static const double kVariance = 0.1

Reply:
> Returns the next representable value of from in the direction of to.

Nextafter is used here to get the next representable value, for example, from  0.57499999999999996 to 0.57500000000000007, so 0.1 just signals the direction to change, and the actual change should be much smaller than it. How about `std::nextafter(from, INFINITY)` to avoid a hard-coded variance?

Reply:
I will try it.